api/src/test/resources/application-rbac-ldap.yml

```diff
@@ -0,0 +1,38 @@
+spring:
+  ldap:
+    base: "cn={0},ou=people,dc=kafbat,dc=io"
+    admin-user: "cn=admin,dc=kafbat,dc=io"
+    admin-password: "StrongPassword123"
+    user-filter-search-base: "dc=kafbat,dc=io"
+    user-filter-search-filter: "(&(uid={0})(objectClass=inetOrgPerson))"
+    group-filter-search-base: "ou=people,dc=kafbat,dc=io" # required for RBAC
+oauth2:
+  ldap:
+    activeDirectory: false
```

Comment:
this can be removed as it should default to false anyway, might be a good thing to catch possible issues like in https://github.com/kafbat/kafka-ui/pull/779/files

Reply:
Ok, you're right, let's remove redundant config settings